docs/todo.rst

```diff
@@ -1,6 +1,9 @@
 .. _todo-label:
 
-**To Do**: These are future improvements to Asterix currenly envisionned (not particularly by order of priority):
+Open issues
+---------------
+
+These are future improvements to Asterix currenly envisionned (not particularly by order of priority):
```

Comment:
Not going to put any into GitHub issues instead? Just checking.

Reply:
not right no, we need to clean it first / add current best issues